.github/workflows/icr-image-build-and-push.yml

```diff
@@ -38,3 +40,12 @@ jobs:
         uses: ./.github/actions/icr-build-and-push-images
         with:
           tag: ${{ github.event.inputs.tag }}
+      - name: Set version from tag
+        if : github.event_name == 'release' && github.event.action == 'published'
+        id: set-version-from-tag
+        run: echo "VERSION=${GITHUB_REF_NAME/v}" >> $GITHUB_OUTPUT
```

Comment:
nit: should we set this in the build and push images action? (just a thought, might make it easier to follow the flow, but I could go either way...)

Reply:
I see you point. Mmm... Right now in the action we have and input called `tag` that can have different values. We are using `latest` and the specific release version as values on it, depending of the action. I would say in this case at least from my point of view I would prefer the reusability that we have that introduce that value in the `action` (totally my opinion).